### ISSUE TITLE
Val and Test accuracy clarification

### ISSUE THREAD

**Issue by savitru10** [mkonuk66/EfficientFER#1]
Hello @mkonuk66  , 
Thank you very much for sharing the code 

I ran the code and found the Val accuracy is 71 and test is 69 

Epoch 14/50
Epoch 14/50 [Train]: 100%|██████████| 449/449 [01:29<00:00,  5.04it/s, Loss=0.2531, Acc=78.13%]
Validation: 100%|██████████| 57/57 [00:09<00:00,  6.08it/s, Loss=0.5612, Acc=71.33%]

Early stopping triggered after 14 epochs. 

Please can you share the best model file ?

Thanks 
Sindhu

**Comment by mkonuk66** [comment]
Hello @savitru10 ,

Thank you very much for testing the code and sharing the results!

You can download the best model file from the link below:
[![Google Drive](https://img.shields.io/badge/Google%20Drive-4285F4?style=for-the-badge&logo=googledrive&logoColor=white)](https://drive.google.com/drive/folders/1FiddxWduJSz7rylvksoLdqkGQR3GFBrx?usp=sharing)

Please note that the results might differ slightly because we used additional training data during our experiments, which could lead to variations in validation and test accuracy.

Thanks again for your interest and feedback!

Best regards,
Mehmet

**Comment by savitru10** [comment]
Dear @mkonuk66  

Thank you very much for sharing the best model .

May i request you to please share what kind of additional training data was used in your experiements ? 
Like was it additional data from other datasets ? ( This is apart from Augmentation techniques )

Thank you 
Best Regards
Sindhu Mohana
